src/main/java/com/easypost/exception/EasyPostException.java

```diff
@@ -8,50 +8,100 @@
 
 package com.easypost.exception;
 
+import com.easypost.model.Error;
+
+import java.util.List;
+
 public class EasyPostException extends Exception {
 
     private static final long serialVersionUID = 1L;
```

Comment:
Reference for future travelers of what this is: https://www.baeldung.com/java-serial-version-uid